.github/workflows/cve-scanning-python.yml

```diff
@@ -0,0 +1,35 @@
+name: CVE Scanning for Python
+
+on:
+  pull_request:
+    paths:
+      - 'python/pyproject.toml'
+      - 'python/safety-policy.yml'
+      - '.github/workflows/cve-scanning-python.yml'
```

Comment:
where are these files? 
I don't see safety-policy.yml.
Pyproject.toml is not under /python. pyproject.toml but at root.

Reply:
Also do we want this to run only when those files are updated, or on every PR? Is it not better to run them on evert PR to check the packages versions more often?

Reply:
@TheJuanAndOnly99 @maoo Could you please advise ?

Reply:
Yes, that is correct. Apologies for the mistake. Please adapt the file path as needed.

Reply:
i will change the files' path accordingly.
this action is going to scan the dependencies only, and the dependencies will changed only when these files get changed, so no need to run the action for every PR, but only when those files got changed.

Reply:
ok so let's say we have a dependency that becomes outdated or vulnerable but we didn't change these files, what's the strategy to detect and correct that? Is there something else separated from this PR dedicated to this purpose?

Reply:
hmm, right. I see what you mean. I agree with you.
However it is not clear what would be the exact behavior from their doc,
_When a pull request contains any files that are not matched by on:pull_request:paths-ignore or on:pull_request:paths, the workflow runs the actions and scans all of the files changed in the pull request_
I understood the scan is still going to run if any files not matched, we could merge and see what happen and update this config accordingly.

Reply:
Good point @symphony-soufiane , I'd suggest to add - at the top of the actions - the following block, which will run the scan at the beginning/end of each working day:

```
on:
  schedule:
    - cron: '0 8,18 * * 1-5'
```

This would cover the case you describe. /cc @TheJuanAndOnly99

Reply:
thx @maoo @symphony-soufiane